packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -103,12 +104,11 @@ function wrapWithRuntimeFactoryIfNeeded(packageJson: IFluidPackage, fluidModule:
                 [defaultFactory.type, Promise.resolve(defaultFactory)],
             ]),
         );
-        return {
-            fluidExport: {
-                IRuntimeFactory: runtimeFactory,
-                IFluidDataStoreFactory: dataStoreFactory,
-            },
+        const fluidExport: IFluidModule["fluidExport"] & FluidObject<IFluidDataStoreFactory> = {
+            IRuntimeFactory: runtimeFactory,
+            IFluidDataStoreFactory: dataStoreFactory,
```

Comment:
Are we able to just drop the `IFluidDataStoreFactory` here and only return a normal `IFluidModule`?

Reply:
yeah. i think we can. nice catch!